website/docs/r/database_secret_backend_static_role.md

```diff
@@ -38,6 +38,16 @@ resource "vault_database_secret_backend_static_role" "static_role" {
   rotation_period     = "3600"
   rotation_statements = ["ALTER USER \"{{name}}\" WITH PASSWORD '{{password}}';"]
 }
+
+resource "vault_database_secret_backend_static_role" "static_role" {
+  backend             = vault_mount.db.path
+  name                = "my-static-role"
```

Comment:
We should make sure the resource name as well as the `name` parameter are distinct from the previous example. Currently they are both named `static_role` and `my-static-role` respectively